src/release/breaking-changes/2-5-deprecations.md

```diff
@@ -0,0 +1,346 @@
+const Form form = Form(autovalidate: false);
+final autoMode = form.autovalidate;
+
+const FormField formField = FormField(autovalidate: true);
+const FormField formField = FormField(autovalidate: false);
+final autoMode = formField.autovalidate;
+
+const TextFormField textFormField = TextFormField(autovalidate: true);
+const TextFormField textFormField = TextFormField(autovalidate: false);
+
+const DropdownButtonFormField dropDownButtonFormField = DropdownButtonFormField(autovalidate: true);
+const DropdownButtonFormField dropdownButtonFormField = DropdownButtonFormField(autovalidate: false);
+```
+
+Code after migration:
+<!-- skip -->
+```dart
+const Form form = Form(autovalidateMode: AutovalidateMode.always);
+const Form form = Form(autovalidateMode: AutovalidateMode.disabled);
+final autoMode = form.autovalidateMode;
+
+const FormField formField = FormField(autovalidateMode: AutovalidateMode.always);
+const FormField formField = FormField(autovalidateMode: AutovalidateMode.disabled);
+final autoMode = formField.autovalidateMode;
+
+const TextFormField textFormField = TextFormField(autovalidateMode: AutovalidateMode.always);
+const TextFormField textFormField = TextFormField(autovalidateMode: AutovalidateMode.disabled);
+
+const DropdownButtonFormField dropDownButtonFormField = DropdownButtonFormField(autovalidateMode: AutovalidateMode.always);
+const DropdownButtonFormField dropdownButtonFormField = DropdownButtonFormField(autovalidateMode: AutovalidateMode.disabled);
+```
+
+[In-depth migration guide available]: /release/breaking-changes/form-field-autovalidation-api
+
+**References**
+
+API documentation:
+* [`Form`][]
+* [`FormField`][]
+* [`TextFormField`][]
+* [`DropdownButtonFormField`][]
+* [`AutovalidateMode`][]
+
+Relevant issues:
+* [Issue 56363]({{site.repo.flutter}}/issues/56363)
+* [Issue 18885]({{site.repo.flutter}}/issues/18885)
+* [Issue 15404]({{site.repo.flutter}}/issues/15404)
+* [Issue 36154]({{site.repo.flutter}}/issues/36154)
+* [Issue 48876]({{site.repo.flutter}}/issues/48876)
+
+Relevant PRs:
+* Deprecated in [#59766]({{site.repo.flutter}}/pull/59766)
+* Removed in [#90292]({{site.repo.flutter}}/pull/90292)
+
+[`Form`]: {{site.api}}/flutter/widgets/Form-class.html
+[`FormField`]: {{site.api}}/flutter/widgets/FormField-class.html
+[`TextFormField`]: {{site.api}}/flutter/material/TextFormField-class.html
+[`DropdownButtonFormField`]: {{site.api}}/flutter/material/DropdownButtonFormField-class.html
+[`AutovalidateMode`]: {{site.api}}/flutter/widgets/AutovalidateMode-class.html
+
+---
+
+### `FloatingHeaderSnapConfiguration.vsync`
+
+Supported by Flutter Fix: no
+
+The `TickerProvider` `vsync` property of `FloatingHeaderSnapConfiguration` was
+deprecated in v1.19.
+
+The `vsync` for the animation should instead be specified using
+`SliverPersistentHeaderDelegate.vsync`.
+
+**Migration guide**
+
+Code before migration:
+<!-- skip -->
+```dart
+// TODO - no original migration guide
```

Comment:
@LongCatIsLooong can you also provide a snippet of before and after code? There is no migration guide form the original change.

Reply:
Oh https://github.com/flutter/flutter/pull/56413 wasn't breaking. I'll try to come up with one.

Reply:
code before change:
```dart
class MyVeryOwnSliverPersistentHeaderDelegate extends SliverPersistentHeaderDelegate {
    FloatingHeaderSnapConfiguration? get snapConfiguration => FloatingHeaderSnapConfiguration(vsync: <Your TickerProvider>);
}
```

after

```dart
class MyVeryOwnSliverPersistentHeaderDelegate extends SliverPersistentHeaderDelegate {
    FloatingHeaderSnapConfiguration? get snapConfiguration => FloatingHeaderSnapConfiguration();
    TickerProvider? get vsync => <Your TickerProvider>;
}
```

Reply:
Thanks! It's ok about adding a separate guide at this point. This should do just fine. :) 
I think in the past we weren't requiring them for deprecations because they weren't breaking at the time they landed. Later when we take the deprecations out is when people feel it.